bigbluebutton-html5/public/locales/en.json

```diff
@@ -51,6 +51,7 @@
     "app.captions.pad.dictationOffDesc": "Turns speech recognition off",
     "app.captions.pad.speechRecognitionStop": "Speech recognition stopped due to the browser incompatibility or some time of silence",
     "app.textInput.sendLabel": "Send",
+    "app.title.defaultViewLabel": "Default",
```

Comment:
Let's add a bit of context - otherwise it could be misunderstood when translating i.e. to default on X